hs_core/views/resource_folder_hierarchy.py

```diff
@@ -101,7 +101,8 @@ def data_store_structure(request):
     )
 
 
-def data_store_folder_zip(request):
+def data_store_folder_zip(request, res_id=None, input_coll_path=None,
+                          output_zip_file_name=None, remove_original_after_zip=None):
     """

```

Comment:
@aphelionz OK. Got it. I am thinking probably it is better to modify this data_store_folder_zip() to check both request.POST and request.data for parameter values then. Something like the following:
```
res_id = request.POST.get('res_id', request.data.get('res_id', None))
```